protocol/0040-ASSF-asset_framework.md

```diff
@@ -212,6 +220,20 @@ NOTE 2: Before running this function, the user must run the ERC-20-standard `app
 
 This section will be expanded if additional ethereum based token standards are supported by Vega. New bridges will be expected to implement `IVega_Bridge`.
 
+#### Built-In Assets
```

Comment:
Asset will either:
- get created by the market proposal (on enactment the ELS asset is created) in that case arbitrary minting is disabled; only the "liquidity mechanism" can mint at epoch end.
- XOR the builtin asset is created by governance in which case the anyone-can-mint function is enabled; "liquidity mechanism" cannot mint them. 

On top of that it is possible to DISABLE creation of builtin assets by governance.